docs/pages/machine-id/guides/jenkins.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Using Teleport Machine ID with Jenkins
+description: Using Teleport Machine ID with Jenkins
+---
+
+Jenkins is an open source automation server that is frequently used to build
+Continuous Integration and Continuous Delivery (CI/CD) pipelines.
+
+In this guide, we will demonstrate how to migrate existing Jenkins pipelines to
+utilize Machine ID with minimal changes.
+
+## Prerequisites
+
+You will need the following tools to use Teleport with Jenkins.
+
+- The Teleport Auth Service and Proxy Service version >= 9.0.0, deployed on
+  your own infrastructure or managed via Teleport Cloud.
+- `ssh` OpenSSH tool
+- Jenkins
+
+## Architecture
+
+Before we begin, it should be noted that Jenkins is a tool that is notoriously
+difficult to secure. Machine ID is one part of securing your infrastructure,
+but it alone is not sufficient. Below we will provide some basic guidance which
+can help improve the security posture of your Jenkins install.
+
+**Single-host deployments:** The simplest Jenkins deployments have the
```

Comment:
```suggestion
### Single-host deployments

The simplest Jenkins deployments have the
```

I know I recommended bolded text here, but I should have previewed the results in my local dev server first. I think the text can still be broken up a bit more. I've suggested using H3s.